make/jdk/src/classes/build/tools/taglet/JSpec.java

```diff
@@ -183,6 +182,15 @@ public String toString(List<? extends DocTree> tags, Element elem) {
         return sb.toString();
     }
 
+    private String currentPath() {
+        try {
+            Class<?> c = Class.forName("jdk.javadoc.internal.doclets.formats.html.HtmlDocletWriter");
+            ThreadLocal<?> tl = (ThreadLocal<?>) c.getField("CURRENT_PATH").get(null);
```

Comment:
I hope the performance overhead of repeated reflective lookup is acceptable... Ideally we should keep the `Field` in a static final variable.

Reply:
Ok, fair point (although n is fairly small). Pushed an update that caches the ThreadLocal in a static field.